pulpcore/pulpcore/app/settings.py

```diff
@@ -36,10 +34,61 @@
     'pulpcore.app.files.HashingFileUploadHandler',
 )
 
+# Dynaconf Configuration
+
+SECRET_KEY = True
+"""Django `check` requires the SECRET_KEY to exist before initialing
+INSTALLED_APPS this value is set to True but will be overwritten after by the
+value exported as PULP_SECRET_KEY or defined in the file PULP_SETTINGS"""
+
+GLOBAL_ENV_FOR_DYNACONF = "PULP"
+"""This defines which environment variable global prefix dynaconf will load
+That means that `export PULP_FOO=1` will be loaded to `django.conf.settings.FOO
+On command line it is possible to check it with `dynaconf list -k foo`
+"""
+
+ENVVAR_FOR_DYNACONF = "PULP_SETTINGS"
+"""This defines which path dynaconf will look to load config files
+example: `export PULP_SETTINGS=/path/to/settings.toml` and the format can be
+.ini, .json, .yaml or .toml
+
+e.g::
```

Comment:
I think these things would be more useful in the docs themselves. This file won't be read by many users in practice since it's installed w/ the pulp code in the python path. Can this be moved to the docs?